src/views/Dashboard/Doctor/Account/Certification.vue

```diff
@@ -188,25 +196,17 @@ export default {
       if (!this.$refs.certificationForm.validate()) {
         return
       }
-      try {
-        this.isLoading = true
-        const certificationInfo = {
-          title: this.certTitle,
-          issuer: this.certIssuer,
-          month: this.certMonth,
-          year: this.certYear,
-          description: this.certDescription,
-          supporting_document: this.certSupportingDocumentsUrl
-        }
-
-        await createCertification(this.api, this.pair, certificationInfo, () => {
-          this.closeCertificationDialog()
-          this.isLoading = false
-        })
-      } catch (err) {
-        console.log(err)
-        this.isLoading = false
+      const certificationInfo = {
+        title: this.certTitle,
+        issuer: this.certIssuer,
+        month: this.certMonth,
+        year: this.certYear,
+        description: this.certDescription,
+        supporting_document: this.certSupportingDocumentsUrl
       }
+      await this.dispatch(createCertification, this.api, this.pair, certificationInfo, () => {
```

Comment:
Apa ini work mas pas `this.closeCertificationDialog()` nya di panggil di sini? just currious 🤔